TestTraceGen/main.py

```diff
@@ -13,7 +13,7 @@ def get_input():
             print("Error in input. Please try again.")
 
 
-def write_traces_to_file(failing_traces: list, passing_traces: list, user_input: int):
+def write_traces_to_file(failing_traces: list, passing_traces: list, user_input: int, file_type: int):
```

Comment:
Jeg er ikke en fan af det her file_type noget. Det ser ud til, at den eneste forskel er filtypen og at den har en ekstra bracket, når det skal være json. Hvorfor skal der være mulighed for at lave txt filer? Og hvorfor kan vi ikke bare bruge json filerne, når forskellen er så minimal?